docs/source/app-dev/explicit-contract-disclosure.rst

```diff
@@ -0,0 +1,94 @@
+.. Copyright (c) 2023 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+.. SPDX-License-Identifier: Apache-2.0
+
+.. _explicit-contract-disclosure:
+
+(Experimental) Explicit contract disclosure
+###########################################
+
+Starting with Canton 2.6, explicit contract disclosure is introduced as an (experimental) feature
```

Comment:
Throughout the doc, I've referred to the artifact as Canton, even though we are in the Daml SDK docs. Double checking that this is fine and we don't need to consider the feature in a more general form, right?